formcentral/server_test.go

```diff
@@ -311,3 +316,61 @@ func TestGetTranslatorReturns500OnTranslationError(t *testing.T) {
 	err := s.GetTranslator(c)
 	assert.Equal(t, err.(*echo.HTTPError).Code, 500)
 }
+
+func TestGetSurveysByPageID(t *testing.T) {
```

Comment:
Overall, looks perfect. But let's test some other paths too -- In particular, just asserting that it gets the right form when there are several (i.e. it gets the latest form before the timestamp). This is the logic we will change soon so it's good to get tests up so we can easily modify them. This will involve putting some of this logic here into helper functions and vars too, I'm sure.

Reply:
_sounds good, see, https://github.com/vlab-research/fly/pull/24/commits/0367d3d7da14b4f4874dc5ff9a9a0c94a5117127_